### PR TITLE
fix: internal router

### PR DESCRIPTION
**Pull request by thisisnithin**
## Motivation and Context

1. Missed a place in changing url for operations client
2. Input variables parse error

## Marketing Changelog

None

#### Checklist

- [x] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: packages/sdk/src/server/operations-client.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class OperationsClient<Queries = any, Mutations = any, Subscriptions = an
 			? Omit<OperationArgs<T, never>, 'input'>
 			: OptionalInput<Subscriptions[T], T>
 	): Promise<AsyncGenerator<Subscriptions[T] extends { response: any } ? Subscriptions[T]['response'] : never>> => {
-		const url = `${this.options.baseURL}/internal/operations/${String(args.operationName)}`;
+		const url = `${this.options.baseURL}/operations/${String(args.operationName)}`;
```
Comment (jensneuse) [inline packages/sdk/src/server/operations-client.ts:92]: This baseURL points to the internal port?

File: pkg/apihandler/apihandler.go [hunk 2]
```diff
@@ -2240,7 +2242,11 @@ func (h *FunctionsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(http.StatusBadRequest)
 			return
 		}
-		ctx.Variables = variablesBuf.Bytes()
+		if h.internal {
```
Comment (jensneuse) [inline pkg/apihandler/apihandler.go:2245]: Looks like a fundamental issue. How did we not see this earlier?

**Review by jensneuse — APPROVED** [review]

File: packages/sdk/src/server/operations-client.ts (hunk 1 above)
Comment (thisisnithin) [reply packages/sdk/src/server/operations-client.ts:92]: Yes

File: pkg/apihandler/apihandler.go (hunk 2 above)
Comment (thisisnithin) [reply pkg/apihandler/apihandler.go:2245]: It was my mistake. I hadn't seen how functions were handled and modified code in the previous PR.

Comment (jensneuse) [reply pkg/apihandler/apihandler.go:2245]: Ok, cool. Ship it. ;)

File: packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+			},
+		});
+		const slowPromise = operations.query({
+			operationName: 'incremental/delay',
+			input: {
+				seconds: 1,
+			},
+		});
+		const fast = await fastPromise;
+		if (fast.errors) {
```
Comment (StarpTech) [inline packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts:18]: What's that?

File: packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import { createOperation } from '../../generated/wundergraph.factory';
+
+export default createOperation.subscription({
+	handler: async function* ({ operations }) {
+		const fastPromise = operations.query({
```
Comment (StarpTech) [inline packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts:5]: Do we need such a complicated test to cover that the operations client is working?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts (hunk 3 above)
Comment (fiam) [reply packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts:18]: @thisisnithin It think you can delete that `if (fast.errors) {` entirely, let it blow up if the operation fails.

**Review by Pagebakers — APPROVED** [review]

File: packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts (hunk 4 above)
Comment (thisisnithin) [reply packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts]: Nope, I was trying a couple of other things. Simplified it.

File: packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts (hunk 3 above)
Comment (thisisnithin) [reply packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts]: Simplified test.

**Review by fiam — APPROVED** [review]
LGTM

**Pull request merged**